gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/factories/GafferHelmValuesFactory.java

```diff
@@ -39,6 +40,7 @@
 public final class GafferHelmValuesFactory {
 
     private static final String KIND = "Gaffer";
+    private static final String DEFAULT_ADMIN_USER = "ADMIN";
```

Comment:
Maybe change `DEFAULT_ADMIN_USER = "ADMIN"` to be `DEFAULT_SYSTEM_USER = "GAAS_SYSTEM_USER"`. This would help separate the `ADMIN` user used for carrying out admin tasks on a crated graph from the GaaS system user used to create the graphs.

Reply:
Done